assets/models/system/databricks-dolly-v2-7b/asset.yaml

```diff
@@ -0,0 +1,4 @@
+extra_config: model.yaml
+spec: spec.yaml
+type: model
+categories: ["Foundation Models"]
```

Comment:
(nit) add a new line at the end of the file

Reply:
Added